web/viewer.js

```diff
@@ -47,31 +47,31 @@ function getViewerConfiguration() {
       customScaleOption: document.getElementById("customScaleOption"),
       previous: document.getElementById("previous"),
       next: document.getElementById("next"),
```

Comment:
Shoudn't the previous and next buttons also be renamed here since they are also part of the toolbar?